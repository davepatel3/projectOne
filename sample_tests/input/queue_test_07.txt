int
6
2
2
df
ps
ps
pf
ef -17
ef 56
db
ps
eb 8
db
gc
gc
ef 10
df
pb
ps
pb
df
ef 75
eb 36
ps
gf
pf
db
gb
pf
pb
ef -60
ef -83
gc
eb 17
df
gs
ef 10
gc
ps
df
ps
ef 40
eb -84
db
eb -97
df
db
ef 94
ef 78
ps
gc
db
pf
ef -75
df
eb 76
db
gc
df
ef -53
qt
