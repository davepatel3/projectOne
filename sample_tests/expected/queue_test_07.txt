cyclic_double_queue is empty!
[-][-][-][-][-][-]
[-][-][-][-][-][-]
cyclic_double_queue is empty!
-17
[-][-][-][-][+][-]
8
6
6
10
[-][-][-][-][-][-]
cyclic_double_queue is empty!
cyclic_double_queue is empty!
[-][-][-][+][+][-]
75
36
cyclic_double_queue is empty!
cyclic_double_queue is empty!
cyclic_double_queue is empty!
6
-83
2
6
[-][-][+][+][+][-]
10
[-][-][-][+][+][-]
-84
40
-97
[-][+][+][+][+][-]
6
17
-75
76
6
94
